tests/pipelines/pixart_sigma/test_pixart.py

```diff
@@ -399,7 +399,7 @@ def test_pixart_1024_without_resolution_binning(self):
     def test_pixart_512_without_resolution_binning(self):
         generator = torch.manual_seed(0)
 
-        transformer = Transformer2DModel.from_pretrained(
```

Comment:
we should have kept this test, can we add it back, and name it  `test_pixart_512_without_resolution_binning_legacy_class` or something like this?
ane make sure to have a similar slow test for dit 

in the future, I think we should always kept the test with the legacy class name, no? cc @DN6